app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -57,8 +64,15 @@ def process_event(event)
     when 'Node'
       event_data[:container_node_name] = event_data[:name]
     when 'Pod'
+      if event_data.key?(:fieldpath)
+        if event_data[:fieldpath].include? 'spec.containers'
+          event_data[:container_name] = event_data[:fieldpath][16..-2]
+          event_data[:kind] = 'Container'
+        end
```

Comment:
This is even bugged I suppose, what do you do if `event_data.key?(:fieldpath)` is present (always) and `include? 'spec.containers'` is false? Is the event lost? (pretty much every pod event)